requirements.txt

```diff
@@ -17,3 +19,4 @@ scp>=0.13.0,<0.14.0
 celery>=4.4.3,<4.5.0
 swapper~=1.1.0
 django-cache-memoize~=0.1
+netaddr~=0.8.0
```

Comment:
is this still used?

Reply:
Yes, it is. `ipaddr` does not provide a quick way to get the next subnet. I wonder if I can just port this functionality and inherit everything else from `ipaddr`. I kept this to work at the last when everything else is ready.

Reply:
it's ok, we use this library in openwisp-monitoring too.